packages/trpc/server/routers/viewer/availability/schedule/getScheduleByUserId.handler.ts

```diff
@@ -0,0 +1,56 @@
+import { DEFAULT_SCHEDULE } from "@calcom/lib/availability";
+
+import { TRPCError } from "@trpc/server";
+
+import type { TrpcSessionUser } from "../../../../trpc";
+import { getHandler } from "./get.handler";
+import type { TGetByUserIdInputSchema } from "./getScheduleByUserId.schema";
+
+type GetOptions = {
+  ctx: {
+    user: NonNullable<TrpcSessionUser>;
+  };
+  input: TGetByUserIdInputSchema;
+};
+
+export const getScheduleByUserIdHandler = async ({ ctx, input }: GetOptions) => {
+  const foundUserDefaultId = await prisma.user.findUnique({
+    where: {
+      id: input.userId,
+    },
+    select: {
+      defaultScheduleId: true,
+    },
+  });
+
+  try {
+    // This looks kinda weird that we throw straight in the catch - its so that we can return a default schedule if the user has not completed onboarding @shiraz will loveme for this
+    if (!foundUserDefaultId?.defaultScheduleId) {
+      throw new TRPCError({
```

Comment:
Don't throw a trpc error here, keep it simple to clarify it'll be caught (or maybe you intended it not to be caught but it does)